tests/integration/test_dbapi_integration.py

```diff
@@ -16,6 +16,7 @@
 import pytest
 import pytz
 import requests
+from tzlocal import get_localzone_name  # type: ignore
```

Comment:
Could you please provide a snippet using `astimezone` to get name/offset timezone?

Reply:
Numeric offsets also work with Trino. You are correct that we cannot use `astimezone` to get named offset.

However after thinking more about it it's possible for the numeric offset to cause incorrect results in a long running application because the offset will be determined once and if after few mins/hours DST change happens then the numeric offset will be incorrect while named offset should work correctly.

So leave it as it is. Thanks.